third_party/bigframes_vendored/pandas/core/arrays/arrow/accessors.py

```diff
@@ -6,6 +6,79 @@
 from bigframes import constants
 
 
+class ListAccessor:
+    """
+    Accessor object for list data properties of the Series values.
+    """
+
+    def len(self):
+        """
+        Return the length of each list in the Series.
+
+        Returns
+        -------
+        Series
+            The length of each list.
+
+        See Also
+        --------
+        str.len : Python built-in function returning the length of an object.
```

Comment:
Also, please review the `See also` format in `Series.case_when`. Of the listed APIs, `StringMethods.len` seems most relevant to our current implementation. Maybe we can just keep this one and remove other two?

Reply:
Done